var/spack/repos/builtin/packages/py-gensim/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyGensim(PythonPackage):
+    """Gensim is a Python library for topic modelling, document indexing and
+       similarity retrieval with large corpora. Target audience is the natural
+       language processing (NLP) and information retrieval (IR) community."""
+
+    homepage = "https://pypi.org/project/gensim/"
+    url      = "https://files.pythonhosted.org/packages/3a/bc/1415be59292a23ff123298b4b46ec4be80b3bfe72c8d188b58ab2653dee4/gensim-3.8.0.tar.gz"
+
+    version('3.8.0', sha256='ec5de7ff2bfa8692fa96a846bb5aae52f267fc322fbbe303c1f042d258af5766')
```

Comment:
It seems 3.8.1 is already out. Can we add that too?

Reply:
Sure thing. Added!